tests/test_http_parser.py

```diff
@@ -1207,8 +1178,8 @@ async def test_http_response_parser_bad_chunked_strict_py(
     text = (
         b"HTTP/1.1 200 OK\r\nTransfer-Encoding: chunked\r\n\r\n5 \r\nabcde\r\n0\r\n\r\n"
     )
-    messages, upgrade, tail = response.feed_data(text)
-    assert isinstance(messages[0][1].exception(), http_exceptions.TransferEncodingError)
+    with pytest.raises(http_exceptions.TransferEncodingError, match="5"):
```

Comment:
Is the entire string `'5'` or just somewhere in it?
```suggestion
    with pytest.raises(http_exceptions.TransferEncodingError, match="^5$"):
```

Reply:
Somewhere... These are parser test, remember. The match needs to rather broad to match both parsers. This is a mild improvement over the existing test which doesn't use the match argument.